From marcel@holtmann.org Sat Oct 29 04:47:56 2005
Subject: USB: Delete leftovers from bluetty driver
From: Marcel Holtmann <marcel@holtmann.org>
To: Greg K-H <greg@kroah.com>
Date: Sat, 29 Oct 2005 13:43:04 +0200
Message-Id: <1130586184.5360.15.camel@blade>

[PATCH] USB: Delete leftovers from bluetty driver

This patch deletes the bluetooth.txt help file of the bluetty driver and
hands over its major device nodes for character devices to the RFCOMM TTY
implementation of the Bluetooth subsystem.

Signed-off-by: Marcel Holtmann <marcel@holtmann.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

 Documentation/devices.txt       |   12 +++++-----
 Documentation/usb/bluetooth.txt |   44 ----------------------------------------
 2 files changed, 6 insertions(+), 50 deletions(-)

--- gregkh-2.6.orig/Documentation/devices.txt	2005-11-02 09:25:58.000000000 -0800
+++ gregkh-2.6/Documentation/devices.txt	2005-11-02 10:28:12.000000000 -0800
@@ -2917,14 +2917,14 @@
 		196 = /dev/dvb/adapter3/video0    first video decoder of fourth card
 
 
-216 char	USB BlueTooth devices
-		  0 = /dev/ttyUB0		First USB BlueTooth device
-		  1 = /dev/ttyUB1		Second USB BlueTooth device
+216 char	Bluetooth RFCOMM TTY devices
+		  0 = /dev/rfcomm0		First Bluetooth RFCOMM TTY device
+		  1 = /dev/rfcomm1		Second Bluetooth RFCOMM TTY device
 		    ...
 
-217 char	USB BlueTooth devices (alternate devices)
-		  0 = /dev/cuub0		Callout device for ttyUB0
-		  1 = /dev/cuub1		Callout device for ttyUB1
+217 char	Bluetooth RFCOMM TTY devices (alternate devices)
+		  0 = /dev/curf0		Callout device for rfcomm0
+		  1 = /dev/curf1		Callout device for rfcomm1
 		    ...
 
 218 char	The Logical Company bus Unibus/Qbus adapters
--- gregkh-2.6.orig/Documentation/usb/bluetooth.txt	2005-08-28 16:41:01.000000000 -0700
+++ /dev/null	1970-01-01 00:00:00.000000000 +0000
@@ -1,44 +0,0 @@
-INTRODUCTION
-
-  The USB Bluetooth driver supports any USB Bluetooth device.
-  It currently works well with the Linux USB Bluetooth stack from Axis 
-  (available at http://developer.axis.com/software/bluetooth/ ) and 
-  has been rumored to work with other Linux USB Bluetooth stacks.
-
-
-CONFIGURATION
-
-  Currently the driver can handle up to 256 different USB Bluetooth 
-  devices at once. 
-
-  If you are not using devfs:
-    The major number that the driver uses is 216 so to use the driver,
-    create the following nodes:
-	mknod /dev/ttyUB0 c 216 0
-	mknod /dev/ttyUB1 c 216 1
-	mknod /dev/ttyUB2 c 216 2
-	mknod /dev/ttyUB3 c 216 3
-		.
-		.
-		.
-	mknod /dev/ttyUB254 c 216 254
-	mknod /dev/ttyUB255 c 216 255
-
-  If you are using devfs:
-    The devices supported by this driver will show up as
-    /dev/usb/ttub/{0,1,...}
-
-  When the device is connected and recognized by the driver, the driver
-  will print to the system log, which node the device has been bound to.
-
-
-CONTACT:
-
-  If anyone has any problems using this driver, please contact me, or 
-  join the Linux-USB mailing list (information on joining the mailing 
-  list, as well as a link to its searchable archive is at 
-  http://www.linux-usb.org/ )
-
-
-Greg Kroah-Hartman
-greg@kroah.com
